units/systemd-nspawn@.service.in

```diff
@@ -28,9 +28,9 @@ Delegate=yes
 TasksMax=16384
 {{SERVICE_WATCHDOG}}
 
-# Enforce a strict device policy, similar to the one nspawn configures when it
-# allocates its own scope unit. Make sure to keep these policies in sync if you
-# change them!
+{# Enforce a strict device policy, similar to the one nspawn configures when it
+ # allocates its own scope unit. Make sure to keep these policies in sync if you
+ # change them! #}
```

Comment:
maybe add a broef comment here that explains the { thing, so that this is not dropped again